Mu2eG4/geom/ExtShieldUpstream_v06.txt

```diff
@@ -156,7 +156,19 @@ vector<double> ExtShieldUpstream.centerType18Box1 = {-3620.6,2780.2,  213.6};
 vector<double> ExtShieldUpstream.centerType19Box1 = {-267.8 ,2952.0,  202.4};
 
 // Remote handling room door
-vector<double> ExtShieldUpstream.centerType20Box1 = {9381 ,-103, -11931};
+vector<double> ExtShieldUpstream.centerType20Box1 = {9381 ,-103, -12077.7};
+
+// To move the from working position (A small gap during remote handling operations) in two directions.
+// To move in Z-direction (East-West) : apply 'OpenDistance' (double) between 0.0 (fully closed) and 6248.4 (fully open).
+// OpenDistance = 314.8; default case (working position)
+// To move in X-direction (towards north) : apply 'FrameGap' (double) between 0.0 (closed) and 254.0 (10" open).
+// FrameGap = 0.0; default case (closed position)
+//NB: Neither OpenDistance nor FrameGap can be supplied a negative value.
+
+
+double ExtShieldUpstream.OpenDistance = 314.8;
+double ExtShieldUpstream.FrameGap = -30.0;
```

Comment:
You committed your test case with illegal data.

Reply:
Provided correct data.

Reply:
THanks.